src/core/propertyTypes.js

```diff
@@ -72,6 +72,9 @@ function assetParse (value) {
   var el;
   var parsedUrl;
 
+  // If an element was provided (e.g. canvas or video), just return it.
+  if (typeof value !== 'string') { return value; }
```

Comment:
**Later on** we're going to need to do something about returning either a string or an element. Property types shouldn't be returning two different types of things.